search/scorer/scorer_term.go

```diff
@@ -117,56 +117,60 @@ func (s *TermQueryScorer) SetQueryNorm(qnorm float64) {
 func (s *TermQueryScorer) Score(ctx *search.SearchContext, termMatch *index.TermFieldDoc) *search.DocumentMatch {
 	var scoreExplanation *search.Explanation
```

Comment:
This can be moved to inside the if s.includeScore || s.options.Explain conditional

Reply:
done! and merging it.